src/components/AssignmentTexter.jsx

```diff
@@ -262,15 +262,17 @@ class AssignmentTexter extends React.Component {
         key="previous"
         onClick={this.handleNavigatePrevious}
         disabled={!this.hasPrevious()}
+        style={{ fontSize: "0.9em" }}
       >
-        <NavigateBeforeIcon />
+        <NavigateBeforeIcon /> PREV
       </IconButton>,
       <IconButton
```

Comment:
Is it possible to convert this to an outlined button? Something like this or similar?
```suggestion
      <Button
          variant="contained"
```